app/Models/User.ts

```diff
@@ -0,0 +1,22 @@
+import { DateTime } from 'luxon'
+import { BaseModel, column } from '@ioc:Adonis/Lucid/Orm'
+
+export default class User extends BaseModel {
+  @column({ isPrimary: true })
+  public id: number
+
+  @column()
+  public userName: string
+
+  @column()
+  public email: string
+
+  @column({ serializeAs: null })
+  public password: string
```

Comment:
Create a hook to hash this password

Reply:
I recreated the user model using the `Adonis Auth`.